CLI panic on listing organizations without passing user ID

<!--
Thanks for submitting a bug report. Please fill the template below,
otherwise we will not be able to process this bug report.
-->

#### Summary
<!-- Summarize the problem in a few sentences: -->

CLI panics when listing organizations without specifying a user ID.

#### Steps to Reproduce
<!-- How can we reproduce the problem? -->

1. Login
2. List organizations with `$ ttn-lw-cli organizations ls`

<!--
Please upload relevant configuration (as .txt).
If you use the command "ttn-lw-stack config", you can redact sensitive config.
-->

#### What do you see now?
<!--
Please paste terminal output, upload logs (as .txt) or upload screenshots.
-->

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1c6f159]

goroutine 1 [running]:
go.thethings.network/lorawan-stack/cmd/ttn-lw-cli/commands.glob..func71(0x2b6a040, 0x2bbef90, 0x0, 0x0, 0x0, 0x0)
	/Users/johan/dev/src/lorawan-stack/cmd/ttn-lw-cli/commands/organizations.go:78 +0x1d9
github.com/spf13/cobra.(*Command).execute(0x2b6a040, 0x2bbef90, 0x0, 0x0, 0x2b6a040, 0x2bbef90)
	/Users/johan/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:762 +0x465
github.com/spf13/cobra.(*Command).ExecuteC(0x2b6c640, 0x0, 0x0, 0x1c814cc)
	/Users/johan/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2ec
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/johan/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
main.main()
	/Users/johan/dev/src/lorawan-stack/cmd/ttn-lw-cli/main.go:27 +0x32
```

#### What do you want to see instead?
<!-- Please add some examples or mock-ups if applicable. -->

No panic, but maybe a nice error that there's no user ID passed (see below)

#### Environment
<!--
Your environment: OS/Browser/Gateway/Device/...? Versions? IDs/EUIs?
Paste the output of "ttn-lw-cli version" or "ttn-lw-stack version" if applicable.
-->

Current `master`

#### How do you propose to implement this?
<!-- Please think about how this could be fixed. -->

`getUserID()` may return `nil` when there's no `user-id` flag set or if there's no single argument.

We either need to fallback to the "current user ID" if nothing is specified, or requiring the user to specify their own user ID on these calls.

If you're in doubt, ask @htdvisser as cli code owner

#### Can you do this yourself and submit a Pull Request?
<!-- You can also @mention experts if you need help with this. -->

Will review

--- a/cmd/ttn-lw-cli/commands/organizations.go
+++ b/cmd/ttn-lw-cli/commands/organizations.go
@@ -75,7 +75,7 @@ var (
 			}
 			limit, page, opt, getTotal := withPagination(cmd.Flags())
 			res, err := ttnpb.NewOrganizationRegistryClient(is).List(ctx, &ttnpb.ListOrganizationsRequest{
-				Collaborator: getUserID(cmd.Flags(), nil).OrganizationOrUserIdentifiers(),
+				Collaborator: getUserID(cmd.Flags(), nil).GetOrganizationOrUserIdentifiers(),
 				FieldMask:    types.FieldMask{Paths: paths},
 				Limit:        limit,
 				Page:         page,
@@ -141,7 +141,7 @@ var (
 		Short:   "Create an organization",
 		RunE: asBulk(func(cmd *cobra.Command, args []string) (err error) {
 			orgID := getOrganizationID(cmd.Flags(), args)
-			collaborator := getUserID(cmd.Flags(), nil).OrganizationOrUserIdentifiers()
+			collaborator := getUserID(cmd.Flags(), nil).GetOrganizationOrUserIdentifiers()
 			if collaborator == nil {
 				return errNoCollaborator
 			}

--- a/cmd/ttn-lw-cli/commands/organizations_access.go
+++ b/cmd/ttn-lw-cli/commands/organizations_access.go
@@ -85,7 +85,7 @@ var (
 			if orgID == nil {
 				return errNoOrganizationID
 			}
-			collaborator := getUserID(cmd.Flags(), nil).OrganizationOrUserIdentifiers()
+			collaborator := getUserID(cmd.Flags(), nil).GetOrganizationOrUserIdentifiers()
 			if collaborator == nil {
 				return errNoCollaborator
 			}
@@ -121,7 +121,7 @@ var (
 			if orgID == nil {
 				return errNoOrganizationID
 			}
-			collaborator := getUserID(cmd.Flags(), nil).OrganizationOrUserIdentifiers()
+			collaborator := getUserID(cmd.Flags(), nil).GetOrganizationOrUserIdentifiers()
 			if collaborator == nil {
 				return errNoCollaborator
 			}

--- a/pkg/ttnpb/identifiers.go
+++ b/pkg/ttnpb/identifiers.go
@@ -55,11 +55,27 @@ func (ids UserIdentifiers) IsZero() bool {
 	return ids.UserID == "" && ids.Email == ""
 }
 
+// GetOrganizationOrUserIdentifiers returns the OrganizationIdentifiers as *OrganizationOrUserIdentifiers.
+func (ids *OrganizationIdentifiers) GetOrganizationOrUserIdentifiers() *OrganizationOrUserIdentifiers {
+	if ids == nil {
+		return nil
+	}
+	return ids.OrganizationOrUserIdentifiers()
+}
+
 // OrganizationOrUserIdentifiers returns the OrganizationIdentifiers as *OrganizationOrUserIdentifiers.
 func (ids OrganizationIdentifiers) OrganizationOrUserIdentifiers() *OrganizationOrUserIdentifiers {
 	return &OrganizationOrUserIdentifiers{Ids: &OrganizationOrUserIdentifiers_OrganizationIDs{
 		OrganizationIDs: &ids,
 	}}
+}
+
+// GetOrganizationOrUserIdentifiers returns the UserIdentifiers as *OrganizationOrUserIdentifiers.
+func (ids *UserIdentifiers) GetOrganizationOrUserIdentifiers() *OrganizationOrUserIdentifiers {
+	if ids == nil {
+		return nil
+	}
+	return ids.OrganizationOrUserIdentifiers()
 }
 
 // OrganizationOrUserIdentifiers returns the UserIdentifiers as *OrganizationOrUserIdentifiers.

